src/Core/RevEng.Core.60/ReverseEngineerScaffolder.cs

```diff
@@ -77,7 +78,7 @@ public SavedModelFiles GenerateDbContext(
 
                 ContextName = code.Identifier(options.ContextClassName),
                 ContextDir = outputContextDir,
-                RootNamespace = null,
+                RootNamespace = rootNameSpace,
```

Comment:
This is the biggest concern I have.  From testing I did on my own project, it seemed to be OK with this, but there could be lots of scenarios that I might have missed.